x-pack/plugins/security_solution/common/endpoint/models/policy_config_helpers.ts

```diff
@@ -106,3 +161,46 @@ const getDisabledWindowsSpecificPopups = (policy: PolicyConfig) => ({
     enabled: false,
   },
 });
+
+/**
+ * Returns the provided with only event collection turned enabled
+ * @param policy
+ */
+export const setPolicyToEventCollectionOnly = (policy: PolicyConfig): PolicyConfig => {
+  const updatedPolicy = disableProtections(policy);
+
+  set(updatedPolicy, 'windows.antivirus_registration.enabled', false);
+
+  return updatedPolicy;
+};
```

Comment:
yeah, I was not sure about that - thus why I created this one to set it explicitly. I am not familiar with what hte requirement might have been for your other PR, but I would check with Caitlyn and if in fact it should have been disabled in the first place, then we can just add it to `disableProtections()`

can we maybe follow up on this in a subsequent PR?

Re: name
Yeah you are right. The only reason I named it this way because the Antivirus entry is not a protection. I can change it to `ensureOnlyEventCollectionIsAllowed()`?? what do you think?

Reply:
thanks for the rename, sounds good!

I've created a follow-up issue for this: https://github.com/elastic/security-team/issues/7323, so we can discuss it during grooming